components/edit-collective/Form.js

```diff
@@ -64,6 +64,8 @@ class EditCollectiveForm extends React.Component {
     router: PropTypes.object, // from withRouter
     intl: PropTypes.object.isRequired, // from injectIntl
     query: PropTypes.object, // passed from Page/Router through index/EditCollective
+    // If true, hides menu and headers
+    contentOnly: PropTypes.bool,
```

Comment:
A great solution to wrap legacy code easily, without moving everything around

Reply:
Thanks. I'm considering we're deleting these components after the settings redesign.